src/container.ts

```diff
@@ -12,11 +12,14 @@ export type ContainerEventType =
     "layout-saved";
 
 export class WindowEventArgs extends EventArgs {
-    public readonly window: ContainerWindow;
+    public readonly window?: ContainerWindow;
```

Comment:
Static events currently don't have a window instance in the args so changed to optional.  This might change at a later time if support is added to get window reference from the windowId for you.